docker/package/tezos_voting_wizard.py

```diff
@@ -161,14 +161,7 @@ class Setup(Setup):
 
     # Check whether the baker_alias account is set up to use ledger
     def check_ledger_use(self):
-        baker_alias = self.config["baker_alias"]
-        address = get_proc_output(
-            f"sudo -u tezos {suppress_warning_text} tezos-client "
-            f"{self.config['tezos_client_options']} show address {baker_alias} --show-secret"
-        )
-        if address.returncode == 0:
-            value_regex = b"(?:" + ledger_regex + b")"
-            return bool(re.match(value_regex, address.stdout))
+        return bool(re.match(ledger_regex.decode(), self.config["baker_key_value"]))
 
     def check_baking_service(self):
```

Comment:
I still think that the current message for when no baking is detected is a bit too long:
```
Looks like the tezos-baking-{net} service isn't running.
If this is a mistake, and you should have a baking instance
running on mainnet on this machine, please check if it is set up.
If it isn't, you can use Tezos Setup Wizard to set it up:
tezos-setup-wizard
```
(_also, it says "mainnet" on the third line no matter what_)

I think it should be simplified to something like:
```
No local baking setup for {net} was detected on this machine.
If you want, you can run the `tezos-setup-wizard` to set one up.
```